app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zeros_if_years_less_than_15() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_should_return_1_humans_years_if_animals_years_equal_15() -> None:
+    assert get_human_age(15, 15) == [1, 1]
+
+
+def test_should_convert_cats_and_dogs_years_into_1_human_age() -> None:
```

Comment:
This test case is similar to the one on line 8. Ensure that each test case covers a unique scenario to avoid redundancy.